.gitignore

```diff
@@ -1,5 +1,5 @@
 *.o
+config.lua
 malware.lua
 vmlinux.h
 .nsrc
-
```

Comment:
we are using `\n` in the EOF on the code base, we can change it, but I wouldn't do this for just this file.. also, not in this patch as well..